test/extended/prometheus/storage.go

```diff
@@ -30,7 +30,7 @@ var _ = g.Describe("[sig-storage][Late] Metrics", func() {
 		var ok bool
 		url, _, bearerToken, ok = helper.LocatePrometheus(oc)
 		if !ok {
-			e2e.Failf("Prometheus could not be located on this cluster, failing prometheus test")
+			g.Skip("Prometheus could not be located on this cluster, skipping prometheus test")
```

Comment:
@jan--f if this check is expected to be gating there should probably be another test checking Prometheus is running for non-microshift installation. @pacevedom can you please sync with the @openshift/storage and discuss alterantives?

Reply:
This change does not skip all storage tests, it skips prometheus tests that require storage. Is that the intention?

Reply:
Yes, the intention is to skip anything related to Prometheus, as in MicroShift there is nothing to store/check metrics.
LocatePrometheus returns false only if there is [no service](https://github.com/openshift/origin/blob/master/test/extended/util/prometheus/helpers.go#L68) for it. Does this happen in any situation in regular OCP/HyperShift?

Reply:
The PR title is then very confusing.

What about the other Prometheus tests in [prometheus.go](https://github.com/openshift/origin/blob/ef6536ba596817bd75acd90b56ebbd9e5d8772d3/test/extended/prometheus/prometheus.go#L92)?

Reply:
There are more `Describe` blocks in the file... Most of the tests are behind `apigroup:config.openshift.io`, but some are always running ([`should report telemetry [Late]`](https://github.com/openshift/origin/blob/ef6536ba596817bd75acd90b56ebbd9e5d8772d3/test/extended/prometheus/prometheus.go#L449))

Anyway, from storage POV, it's OK to skip `[sig-storage][Late] Metrics` tests when running without any cloud provider. It's mostly for Azure.

/lgtm